server/configs/application.properties

```diff
@@ -126,27 +126,29 @@ management.server.port=@@shutdownPort@@
 #jsonaccesslog.condition-unless=attributeName
 
 ## START OF CSP ENFORCE BLOCK (DO NOT CHANGE THIS TEXT)
+## CSP Version: e10
 #useLocalBuild#csp.enforce=\
-#useLocalBuild#    default-src 'self' https: ;\
-#useLocalBuild#    connect-src 'self' ${CONNECTION.SOURCES} ;\
-#useLocalBuild#    object-src 'none' ;\
-#useLocalBuild#    style-src 'self' https: 'unsafe-inline' ${STYLE.SOURCES} ;\
-#useLocalBuild#    img-src 'self' https: data: ;\
-#useLocalBuild#    font-src 'self' data: ${FONT.SOURCES} ;\
-#useLocalBuild#    script-src 'self' 'unsafe-eval' 'strict-dynamic' 'nonce-${REQUEST.SCRIPT.NONCE}' ;\
-#useLocalBuild#    base-uri 'self' ;\
-#useLocalBuild#    frame-ancestors 'self' ;\
-#useLocalBuild#    frame-src 'self' ${FRAME.SOURCES} ;\
+#useLocalBuild#    default-src 'self' https: ; \
+#useLocalBuild#    connect-src 'self' ${CONNECTION.SOURCES} ; \
+#useLocalBuild#    object-src 'none' ; \
+#useLocalBuild#    style-src 'self' https: 'unsafe-inline' ${STYLE.SOURCES} ; \
```

Comment:
Can you remind me when we can remove `https:` on a few of these directives? Once we have the admin UI?

Reply:
Nevermind. They're not in the report-only version that will eventually be promoted.